packages/luxon/src/luxon-utils.ts

```diff
@@ -34,9 +34,9 @@ export default class LuxonUtils implements IUtils<DateTime> {
 
   constructor({
     locale,
-    formats
+    formats,
   }: { formats?: Partial<DateIOFormats>; locale?: string } = {}) {
-    this.locale = locale || "en";
+    this.locale = locale || "en-US";
```

Comment:
Because the default for luxon is `en-US` not en, adding the handling to pass through the locale to parse was actually breaking one of the non locale aware parses tests because it was changing it from `en-US` to `en`.  Using `en-US` should ensure that the default behavior doesn't change at all